src/hooks/useIssuesData.tsx

```diff
@@ -1,12 +1,12 @@
-import { Node } from "typescript";
 import { useIssuesContext } from "../contexts/IssuesContext";
 
 export const useIssuesData = () => {
-  const { setIssuesData } = useIssuesContext();
-  const fetchIssues = (inputValue: string) => {
+  const { setIssuesData, setSpinner } = useIssuesContext();
+  const fetchIssues = async (inputValue: string) => {
```

Comment:
argument shouldn't be called `inputValue`, but rather `url` or something like that. `inputValue` in here isn't suitable because it's not necessarily it